src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -61,7 +61,23 @@ public final class FuzzTargetRunner {
   private static final String OPENTEST4J_TEST_ABORTED_EXCEPTION =
       "org.opentest4j.TestAbortedException";
 
-  private static final Unsafe UNSAFE = UnsafeProvider.getUnsafe();
+  private static final Unsafe UNSAFE;
+
+  static {
+    Unsafe unsafe;
+    try {
+      unsafe = UnsafeProvider.getUnsafe();
+    } catch (Throwable t) {
```

Comment:
Is catching `Throwable` really necessary or would `Exception` suffice? It's probably not an issue, but catching `Throwable` always seems extensive.

Reply:
Actually, now that I look at this again, `getUnsafe` doesn't throw anything as `getUnsafeInternal` is called eagerly. I simplified the exception handling accordingly.